ruby_on_rails/app_initialisation.md

```diff
@@ -1,32 +1,42 @@
 # Initialise the Rails App
 
-* [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not. You can do this with `gem update rails`.
+* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v` and update it if you are not.
+Run `rbenv install -s` to check your installed ruby versions.
+Follow the instructions on screen to eventually install it.
 
-* Start a new Rails project using the usual `rails new -d postgresql [project-name]` where the `project-name` is exactly the one you chose before.
+* Run `gem install bundler` to install Bundler.
+You may have it already installed, but this command will make sure that you have the latest version.
+
+* [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
+You can do this with `gem update rails`.
+
+* Start a new Rails project using `rails new [project-name] -d postgresql --skip-sprockets --webpack` where the `project-name` is exactly the one you chose before.
```

Comment:
biggest change of this PR

Reply:
My idea would be to leave the option to use minitest, that's why rspec is not a mandatory step.
I will open a specific issue to make rspec mandatory. see #152